### DISCUSSION TITLE
Accretion disc in the model?

### DISCUSSION THREAD

**Discussion by ayozeah** [phoebe-project/phoebe2#523] [Questions]
Hi! I am working on light curve modelling of X-ray binaries and cataclysmic variables. I have employed the XRBinary code by E. L. Robinson, but I would like to migrate to Phoebe given its capabilities and its Python integration. Is there any option to include an accretion disc in the model? I have looked for this in the documentation but I haven't found anything. Thank you in advance.

**Comment by aprsa** [comment]
Nope, no support for accretion disks at the moment. It's in the pipeline for 2022 though.

**Comment by rpoleski** [comment]
Hi! Are the accretion discs still part of the plan for 2022?

**Reply by kecnry** [reply]
Unfortunately there has not been any active development on accretion disks, yet.  It's still on our wishlist to get around to someday, but we can't promise any timeline at this point.  If/when we do get to it, we'll try to reach out to see if there are any suggestions/requests for particular physics or parameterizations.

-Kyle
